lsp-eslint.el

```diff
@@ -178,7 +188,9 @@
                     (with-current-buffer buffer
                       (list :validate "probe"
                             :packageManager lsp-eslint-package-manager
-                            :codeActionOnSave t
+                            :codeActionOnSave
+                            (list :enable t
```

Comment:
nit: can you move this `(list ...)` to same line with `:codeActionOnSave`?
That would make it easier to distinguish (and the line is not that long)